idpay_virtuemart/idpay_virtuemart.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <extension version="3.6" type="plugin" group="vmpayment" method="upgrade">
-    <name>IDPay for Virtuemart</name>
+    <name>System - IDPay for Virtuemart</name>
     <version>2.0.0</version>
     <description>IDPay payment gateway for Virtuemart</description>
     <author>JMDMahdi, Vispa</author>
```

Comment:
Add your name as developer:

Developer: JMDMahdi, vispa, mnbp1371 , Publisher: IDPay

Reply:
Add Author mail to xml